### ISSUE TITLE
Faktory's inline mode in development or test.

### ISSUE THREAD

**Issue by adrianotadao** [contribsys/faktory_worker_go#39]
Hi there.

I'm using a fake TCP server as Faktory does, just to make my tests green but now I need my jobs running "for real". 

Is there such a thing as "Faktory's inline mode"? 😄

Thank you very much!

**Comment by mperham** [comment]
Yeah, I definitely would like to see some test helpers, like a testContext builder. Go isn’t dynamic enough to support inline mode.
